ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -36,3 +36,49 @@ access_lists:
       - sequence: <int>
         action: <str>
 ```
+
+## IP Community Lists
+
+### Description
+
+AVD supports 2 different data models for community lists:
+
+- The legacy `community_lists` data model that can be used for compatibility with the existing deployments.
+- The improved `ip_community_lists` data model.
+
+Both data models can coexist without conflicts, as different keys are used: `community_lists` vs `ip_community_lists`.
+Community list names must be unique.
+
+The improved data model has a better design documented below:
+
+communities and regexp MUST not be configured together in the same entry
+possible community strings are (case insensitive):
+ - GSHUT
+ - internet
+ - local-as
+ - no-advertise
+ - no-export
+ - <1-4294967040>
+ - aa:nn
+
+### Variables
+
+| Variable | Type | Required | Default | Value Restrictions | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| [<samp>ip_community_lists</samp>](## "ip_community_lists") | List, items: Dictionary |  |  |  | IP Community Lists |
+| [<samp>&nbsp;&nbsp;- name</samp>](## "ip_community_lists.[].name") | String | Required, Unique |  |  | IP Community-list Name |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;entries</samp>](## "ip_community_lists.[].entries") | List, items: Dictionary | Required |  |  |  |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- action</samp>](## "ip_community_lists.[].entries.[].action") | String | Required |  | Valid Values:<br>- permit<br>- deny | "< permit or deny >"<br>Example: "permit"<br> |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;communities</samp>](## "ip_community_lists.[].entries.[].communities") | List |  |  |  | [ "< a_community as string >", "< another_community as string >", ... ]<br>This is optional, if defined - standard community list will be configured<br> |
+| [<samp>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;regexp</samp>](## "ip_community_lists.[].entries.[].regexp") | String |  |  |  | "< regular expression >"<br>If defined, regex community list will be configured |
+
+### YAML
+
+```yaml
+ip_community_lists:
+  - name: <str>
+    entries:
+      - action: <str>
+        communities:
```

Comment:
is it expected like this? the type is not visible here

Reply:
it is not handled correctly no. The issue is that you have not created the schema for the list items. Please update that, and then you will see this correctly.